cves/kernel/CVE-2021-27363.yml

```diff
@@ -55,7 +55,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A pointer leak affected the entire system where an attacker gain access
+  to sensitive informatio through accessing files within the file system. Specific 
+  methods leaks information when called in the system.
```

Comment:
There are a few typos in here, but I also just don't quite get what you're trying to say. What kind of information was leaked? It sounds like everything was leaked but it'd be nice if it was a bit clearer.